Apps/Playground/Android/app/CMakeLists.txt

```diff
@@ -24,6 +24,16 @@ set(BABYLON_NATIVE_PLATFORM "Android")
 set(CMAKE_CXX_EXTENSIONS OFF)
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++17")
 
+if (${ANDROID_JSENGINE_LIBNAME} STREQUAL "jsc")
+    set(NAPI_JAVASCRIPT_ENGINE "JavaScriptCore" CACHE STRING "JavaScript engine for N-API.")
+else()
+    set(NAPI_JAVASCRIPT_ENGINE "V8" CACHE STRING "JavaScript engine for N-API.")
+endif()
```

Comment:
nit: might be better to have an else case that throws an error if the libname is not jsc or v8android.